compiler/src/scala/quoted/runtime/impl/QuotesImpl.scala

```diff
@@ -2481,6 +2482,21 @@ class QuotesImpl private (using val ctx: Context) extends Quotes, QuoteUnpickler
         for sym <- decls(cls) do cls.enter(sym)
         cls
 
+      def newModule(owner: Symbol, name: String, modFlags: Flags, clsFlags: Flags, parents: List[TypeRepr], decls: Symbol => List[Symbol], privateWithin: Symbol): Symbol =
+        assert(parents.nonEmpty && !parents.head.typeSymbol.is(dotc.core.Flags.Trait), "First parent must be a class")
+        val mod = dotc.core.Symbols.newCompleteModuleSymbol(
+          owner,
+          name.toTermName,
+          modFlags | Flags.Final | Flags.Lazy | Flags.Module,
+          clsFlags | Flags.Final | Flags.Module,
+          parents.asInstanceOf, // FIXME
```

Comment:
It's better to make the type parameter of the `asInstanceOf` explicit to avoid confusion, also the FIXME should be expanded to explain what needs to be fixed.

Reply:
Forgot to take care of this one. I added `Symbols.newNormalizedModuleSymbol` to make this work the same way as `newClass`.